Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN find -type d -name '.git' -exec rm -r {} + && \
 
 # Stage 2
 # Run webpack
-FROM node:12 AS webpack
+FROM node:lts AS webpack
```

Comment:
We should use a specific version of node so that it doesn't change without our awareness.